process-compose/mcr-client/process-compose.test.yml

```diff
@@ -7,7 +7,7 @@ processes:
   mcr-client-tests:
     command: |
       set -e
-      cargo test -p mcr-settlement-client --features="e2e" -- --nocapture
+      cargo run --bin test-mcr-settlement-client
```

Comment:
This commit is unrelated to the timestamps. Was it added to fix the CI for this PR?

Reply:
Yeah, CI fix.